composer.json

```diff
@@ -41,7 +42,7 @@
 		"wp-coding-standards/wpcs": "2.3.0"
 	},
 	"scripts": {
-		"php:compatibility": "vendor/bin/phpcs -p -s --runtime-set testVersion '5.6-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
+		"php:compatibility": "vendor/bin/phpcs -p -s --runtime-set testVersion '5.6-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor,packages/*/wordpress --extensions=php",
```

Comment:
You should be able to get rid of this change now.

Reply:
If I remove this and the `wordpress` directory created by `WorDBless` is at the root of this package, then this command fails because it scans the `wordpress` directory. I can off course remove this if it makes sense for the process on Travis.